xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -641,6 +642,7 @@ void DX::DeviceResources::ResizeBuffers()
   else // Otherwise, create a new one using the same adapter as the existing Direct3D device.
   {
     HDR_STATUS hdrStatus = CWIN32Util::GetWindowsHDRStatus();
+    CWinRtUtil::GetWindowsHDRStatus();
```

Comment:
This is necessary?   It's not called from inside CWIN32Util::GetWindowsHDRStatus(); when supported?

Not used return value?

Reply:
that's a leftover from an earlier iteration of the PR, will be removed.